views/overlays/umsk/game16by9.hbs

```diff
@@ -25,6 +25,28 @@
         document.getElementById('rplayer').innerText = data.rplayer || "";
         document.getElementById('lscore').innerText = data.lscore || "";
         document.getElementById('rscore').innerText = data.rscore || "";
+
+        if(data.lCharacter) {
+          var leftIcon = document.createElement('img');
+          leftIcon.id = 'leftCharacter';
+          leftIcon.className = data.lCharacter;
+          leftIcon.style.zoom = '200%';
+          //src is required to remove the dumb border
+          leftIcon.src = 'http://upload.wikimedia.org/wikipedia/commons/c/ce/Transparent.gif';
+          document.getElementById('lplayer').appendChild(leftIcon);
```

Comment:
Hmm, I'm not sure I understand. So if you have an image tag in HTML, then change the class of that image tag whenever the character icon dropdown changes, that doesn't work?